### ISSUE TITLE
Issue about slot

### ISSUE THREAD

**Issue by oyexiaozhi** [HcySunYang/vuese#72]
**Describe the bug**
<template>
  <div>
    <template #searchSlot>
      
      <slot name="searchSlot"></slot>
    </template>
  </div>
</template>

**Expected behavior**
render out the slot in markdown file

**Which version do you use:**
"@vuese/markdown-render": "^2.2.7"
"@vuese/parser": "^2.1.6"

**Comment by oyexiaozhi** [comment]
**Describe the bug**
<template>
  <div>
    <template #searchSlot>
      
      <slot name="searchSlot"></slot>
    </template>
  </div>
</template>

**Comment by oyexiaozhi** [comment]
"<template>
  <div>
    <template #searchSlot>
      
      <slot name="searchSlot"></slot>
    </template>
  </div>
</template>"

**Comment by oyexiaozhi** [comment]
```html
<template>
  <div>
    <template #searchSlot>
      <!-- search slot -->
      <slot name="searchSlot"></slot>
    </template>
  </div>
</template>
```

**Comment by MisicDemone** [comment]
same issue🤕

**Comment by IWANABETHATGUY** [comment]
Duplicate of #119

**Comment by IWANABETHATGUY** [comment]
will be fixed by #161
